src/core/server/usage_collection/index.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export { UsageCollectionServiceConfigType, config } from './config';
+export {
+  CollectorOptions as UsageCollectionCollectorOptions,
+  CollectorFormatForBulkUpload as UsageCollectionCollectorFormatForBulkUpload,
+  Collector as UsageCollectionCollector,
+  Purify as UsageCollectionPurify,
+  UsageCollector as UsageCollectionUsageCollector,
+  MakeSchemaFrom as UsageCollectionMakeSchemaFrom,
+  AllowedSchemaTypes as UsageCollectionAllowedSchemaTypes,
+} from './collectors';
```

Comment:
Renaming all interfaces when exposing them publicly to match the prefix-pattern in core.